ComfortZoneApp/app/src/main/java/com/example/comfortzone/Fragments/InputFragment.java

```diff
@@ -185,6 +186,13 @@ public void onSwipedLeft(int position) {
                 TodayEntry todayEntry = entries.get(position);
                 adapter.remove(position);
                 ComfortLevelEntry comfortEntry = todayEntry.getComfortLevelEntry();
+                LevelsTracker tracker = null;
+                try {
+                    tracker = comfortEntry.getLevelTracker();
+                } catch (ParseException e) {
+                    e.printStackTrace();
+                }
+                tracker.removeEntry(comfortEntry);
```

Comment:
I wonder what happened if you don't do this, but just delete the `comfortEntry` only. i.e. since pointer no longer pointing at anything, will the entry be remove from the list in tracker?

Reply:
it will occupy space in the list, but point to nothing :(